packages/flutter/lib/src/cupertino/checkbox.dart

```diff
@@ -293,16 +467,26 @@ class _CheckboxPainter extends ToggleablePainter {
     notifyListeners();
   }
 
-  BorderSide? get side => _side;
+  BorderSide get side => _side!;
```

Comment:
This seems like it might not be safe.

Reply:
`effectiveBorderSide` is guaranteed to not be null, so by this point `side` is guaranteed to not be null.